### PR TITLE
Add CircleCI E2E tests

### PR DESCRIPTION
**Pull request by inodb**
Run e2e tests from frontend repo against cbioportal backend

### CODE REVIEW

**Comment by inodb** [comment]
Still getting ~15 errors: https://circleci.com/gh/inodb/cbioportal/32

File: portal/pom.xml [hunk 1]
```diff
@@ -105,9 +105,9 @@
             <configuration>
               <artifactItems>
                 <artifactItem>
-                  <groupId>com.github.cbioportal</groupId>
+                  <groupId>com.github.inodb</groupId>
```
Comment (adamabeshouse) [inline portal/pom.xml]: make sure to revert this

**Comment by adamabeshouse** [comment]
cool, so this will run based on master branch?

**Comment by inodb** [comment]
@adamabeshouse it will run the backend in the PR with the frontend defined in the pom against the e2e tests in the same frontend

**Comment by jjgao** [comment]
@inodb what's the plan on this?

**Comment by inodb** [comment]
@jjgao 	I think we should close this for now and revisit where these tests should live. We also want to run them before data updates to the portal. We should come up with some interface that makes it a little easier to run the tests without pulling the entire frontend repo imo. E.g. some docker compose file that runs the whole suite of tests

**Pull request closed without merging**